### DISCUSSION TITLE
I want to delete my GitHub account

### DISCUSSION THREAD

**Discussion by Arshi14tech** [community/community#142755] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Please help me to delete my GitHub account

**Comment by JoaoCoelho2003** [comment, marked as answer]
Go to your Account Settings:

- Click your profile picture in the top-right corner.
-  Select Settings.

Once you're in the Settings page, you'll see a sidebar on the left-hand side. Click on the "Account" option in the sidebar.
This will take you to the page where you can manage your account settings, including deleting your account.
Finally scroll down to the bottom and you will see in red the option to delete your account.

**Reply by jalilahmad45** [reply]
این مراحل را طی کردم حذف نمیشه

**Reply by jalilahmad45** [reply]
کمکم کنید

**Reply by ghost** [reply]
Hi,
I tried to delete with mentioned above navigation but i'm unable to delete the account.Kindly help me with account deletion.
Thanks,
Naveen

**Comment by ghost** [comment]
I have no profile picture on the top RIGHT side of the page in fact everyone has a profile pic on the top left.

**Comment by AmanRai8** [comment]
1. Go to Settings:

    ->Top-right corner → Click on your profile picture → Settings

2. In the left sidebar, scroll to the bottom and click “Account”.

3. Scroll down to “Delete account” section.

4. Click “Delete your account”.

5. You'll be asked to:

     ->Enter your username to confirm.

     ->Possibly enter your password and 2FA code (if enabled).

6. Confirm, and your account will be deleted.

note: If you are an owner of any organization, you will have to remove yourself, transfer ownership, or delete the organization before you can delete your github account

**Reply by jfell8925-boop** [reply]
[](url)delete organization

**Comment by 927625bec154-dotcom** [comment]
I want to delete my GitHub account while deleting I am unfortunately navigate out 
That delete option not showing now what to do now for delete my account
